diambra/arena/make_env.py

```diff
@@ -58,13 +59,16 @@ def env_settings_check(env_settings):
 
 
 def make(game_id, env_settings={}, wrappers_settings={},
-         traj_rec_settings=None, seed=None, rank=0):
+         traj_rec_settings=None, seed=None, rank=0, log_level=logging.INFO):
     """
     Create a wrapped environment.
     :param seed: (int) the initial seed for RNG
     :param wrappers_settings: (dict) the parameters for envWrapping function
+    :param log_level: (int) the logging level (e.g logging.DEBUG)
     """
 
+    logging.basicConfig(level=log_level)
```

Comment:
@discordianfish  is this log level passed also to the arena classes instantiated by the make method?

Reply:
Yes